.gitignore

```diff
@@ -2,3 +2,4 @@
 _build
 *.install
 .merlin
+_opam
```

Comment:
could you add a newline at the end of the line? (its absence shows up as a weird red circle here in the diff, and is also warned about in the `git` command-line diff)

Reply:
Indeed, it is good to configure your editor to always save files with a newline, in #56 a few files are also missing one.

Reply:
Interesting, I've never realised that 😅 @ejgallego thanks for the suggestion, I configured VSCode to always insert a newline :)